packages/dx-react-scheduler/docs/guides/grouping.md

```diff
@@ -24,6 +24,12 @@ A single appointment can appear in multiple groups if it is assigned to [multipl
 
 .embedded-demo({ "path": "scheduler-grouping/basic", "showThemeSelector": true })
 
+## Grouping by Date
+
+The example below shows how to group appointments by date and then by resource. To group by date, use the `GroupingState`'s `groupByDate` property. You can change this order by clicking "Group by Date First" above the Scheduler.
```

Comment:
In the **following demo**, you can change this order by clicking "Group by Date First" above the Scheduler.